packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -550,87 +602,101 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
   const variables: CSSOutput['variables'] = [];
 
   // Quasis are the string pieces of the template literal - the parts around the interpolations
-  const literalResult = node.quasis.reduce<string>((acc, quasi, index): string => {
-    const nodeExpression = node.expressions[index] as t.Expression | undefined;
-
-    if (
-      !nodeExpression ||
-      (t.isArrowFunctionExpression(nodeExpression) && t.isLogicalExpression(nodeExpression.body))
-    ) {
-      const suffix = meta.context === 'keyframes' ? '' : ';';
-      return acc + quasi.value.raw + suffix;
-    }
+  const literalResult = node.quasis.reduce<string>(
+    (acc: string, quasi: t.TemplateLitteral, index: number): string => {
+      const nodeExpression = node.expressions[index] as t.Expression | undefined;
+
+      // Deal with any negative values such as:
+      // margin: -${gridSize}, top: -${gridSize} etc.
+      if (quasi.value.raw.endsWith('-') && !t.isArrowFunctionExpression(nodeExpression)) {
```

Comment:
If this cannot be placed after https://github.com/atlassian-labs/compiled/blob/master/packages/babel-plugin/src/utils/css-builders.tsx#L563, you may want to check whether nodeExpression exists

Reply:
I wonder if we handle `margin: -1 * ${gridsSize}px;` right now, if not do we want to potentially handle it in another issue?

Reply:
@najjiatlassian we actually discussed that workaround but thought that it was less performant of changing the evaluation at build time

Reply:
@at-nathan

The original workaround was:

`margin-left: calc(${gridSize} * - 1px);` 

which did _work_ - but kinda felt a bit hacky (& as @pancaspe87  mentioned - potentially less performant) - the route of modifying the AST directly to become `margin-left: ${-gridSize}px` felt a bit cleaner all around

>If this cannot be placed after https://github.com/atlassian-labs/compiled/blob/master/packages/babel-plugin/src/utils/css-builders.tsx#L563, you may want to check whether nodeExpression exists

Yep - I've moved this will push this with other fixes listed on this PR in the next short while